src/extra/AutoUi/schemaOps.ts

```diff
@@ -0,0 +1,93 @@
+import { JSONSchema7 as JSONSchema } from 'json-schema';
+import $pick from 'lodash/pick';
+import { Dictionary } from '../../unstable-temp/DownloadImageModal/models';
```

Comment:
We also have a common-types.ts. Maybe it would make sense to move the Dictionary definition there and use it all places.